libcodechecker/server/client_db_access_handler.py

```diff
@@ -182,6 +322,16 @@ def getRunData(self, run_name_filter):
             q = q.outerjoin(stmt, Run.id == stmt.c.run_id) \
                 .order_by(Run.date)
 
+            status_q = session.query(Report.run_id,
+                                     Report.detection_status,
+                                     func.count(literal_column('*'))
+                                         .label('status_count')) \
+                .group_by(Report.run_id, Report.detection_status)
+
+            status_sum = defaultdict(defaultdict)
+            for run_id, status, count in status_q:
+                status_sum[run_id][detection_status_enum(status)] = count
```

Comment:
It depends on the DB system. As far as I know SQLite stores enums as strings and PostgreSQL stores them as integers. The common behavior is that when reading the value then it is returned in a string representation.